### DISCUSSION TITLE
Config Syncer Install

### DISCUSSION THREAD

**Discussion by giscus** [appscode/community#62] [Stash]
# Config Syncer Install

New to Config Syncer? Please start here.
Installation Guide Config Syncer operator can be installed via a script or as a Helm chart.
Get a Free License Download a FREE license from AppsCode License Server.
Config Syncer licensing process has been designed to work with CI/CD workflow. You can automatically obtain a license from your CI/CD pipeline by following the guide from here.
Install Helm 3 (Recommended) YAML Using Helm 3 Config Syncer can be installed via Helm using the chart from AppsCode Charts Repository.

https://config-syncer.com/docs/v0.14.6/setup/install/

**Comment by mmckane** [comment]
I am confused. is Config-Syncer open source software or not, this seems to indicated a license is required? If licenses are required what features are paywalled? Also it appears that all previous versions no longer work as the docker.io/appscode/kubed:<version> images no longer exist. I also can no longer open issues in github for this open source project.

**Reply by brokenjacobs** [reply]
I went through updating our entire setup to use the new OCI artifacts for config-syncer with the new helm chart, only to be left with imagepullbackoff errors because it's not deploying the 'enterprise' version? So I'm left with broken clusters that can't pull the old images anymore. 

TLDR; If you ended up here just go replace this with emberstack reflector:
https://github.com/emberstack/kubernetes-reflector

**Reply by brokenjacobs** [reply]
All I can say is thank god we noticed before our certificates expired.

**Comment by herself** [comment]
Yeah, same here. Going with https://github.com/mittwald/kubernetes-replicator to replace it.

I know for sure I will be very wary of using anything AppsCode, you don't just delete images without any announcement to make people get your enterprise version...

**Comment by adriaanmolendijk-tomtom** [comment]
There are consumers of the software that have production workloads running, which could include services that have SLAs or are life saving. Removing existing artifacts breaking running software is completely irresponsible.

**Comment by marcelser** [comment]
breaking existing functionality by even removing the existing images is really an absolute no-go, you probably broke multiple production workloads. I hope everyone moves away from this product to a company that doesn't act like this. With this move they lost all credibility in my eyes. I would never ever pay a dime to them.

**Comment by 8Qbit** [comment]
It was very annoying to notice that the reason our pipelines started to fail was because of this change. Also like broken mentioned it was good to notice this before our certs would have started failing too.

Quick fix was to install the free license but long time solution will be to change something more trustworthy.

Big thumbs down for this.

**Comment by qwrobins** [comment]
Hard to believe that engineers would screw over other engineers this badly. You guys have destroyed any credibility that AppsCode had. There is nothing wrong with wanting to monetize your work, but rug pulling your code base with no announcement and leaving teams scrambling is completely unacceptable and you're seriously charging $1,250 per year for something that has free alternatives that honestly work better and have more features on top of it? You guys are delusional and it wouldn't surprise me if you guys ended up getting sued out of existence.

**Comment by cdtzabra** [comment]
Same thing, a kubed deployment error, `imagepullbackoff` , I look on dockerhub, the image `docker.io/appscode/kubed`  **doesn't exist anymore**.

So I decide to migrate to the new config-syncer with OCI. 

And there again, bad surprises: 
* `imagepullbackoff` again because not the "enterprise" version
* license introduction - I wanted to test with the free license, but the form asks for the "clusterID" - no question for me!

It's unacceptable for appscode to do this kind of thing without any prior communication.

**Comment by krumware** [comment]
For folks brought here by the missing image. The Rancher folks kept one up: https://hub.docker.com/r/rancher/mirrored-appscode-kubed/tags
